File: backend/lostandfound/src/main/java/edu/cit/tiongzon/lostandfound/feature/claims/ClaimDTO.java
package edu.cit.tiongzon.lostandfound.feature.claims;

import lombok.Data;

import java.time.LocalDateTime;

@Data
public class ClaimDTO {
    private Long id;
    private Long itemId;
    private String itemTitle;
    private String itemStatus;
    private Long itemReporterId;
    private Long claimantId;
    private String claimantName;
    private String proofDescription;
    private String proofImagePath;
    private Claim.ClaimStatus status;
    private Claim.PaymentStatus paymentStatus;
    private String paymentIntentId;
    private LocalDateTime createdAt;
    private LocalDateTime lastUpdate;
}

File: mobile/app/src/main/java/edu/cit/tiongzon/lostandfound/feature/claims/data/model/ClaimDto.kt
package edu.cit.tiongzon.lostandfound.feature.claims.data.model

data class ClaimDto(
    val id: Long? = null,
    val itemId: Long? = null,
    val itemTitle: String? = null,
    val itemStatus: String? = null,
    val itemReporterId: Long? = null,
    val claimantId: Long? = null,
    val claimantName: String? = null,
    val proofDescription: String = "",
    val proofImagePath: String? = null,
    val status: String? = null,
    val paymentStatus: String? = null,
    val paymentIntentId: String? = null,
    val createdAt: String? = null,
    val lastUpdate: String? = null
)
